roles/ckan/templates/kubernetes/ckan.yaml

```diff
@@ -225,6 +228,17 @@ spec:
         - name: ckan-resources
           persistentVolumeClaim:
             claimName: ckan-resources-blob
+{% else %}
```

Comment:
@cooper667 could you help me understand why the claim name is different between azurea nad local (and what about AWS?) ckan-storage-blob vs ckan-storage ? Is this if-else here sensible way to solve that?

Reply:
It should both be the same. will merge when we actually have a staging env and remove the difference